docs/en/reference/events.rst

```diff
@@ -52,366 +52,12 @@ methods must be named like the events itself.
 
     <?php
     $evm = new EventManager();
-    $eventName = Events::onSchemaCreateTable;
+    $eventName = Events::onSomething;
     $evm->addEventListener($eventName, new MyEventListener());
 
 .. code-block:: php
 
     <?php
     $evm = new EventManager();
-    $eventNames = [Events::onSchemaCreateTable, Events::onSchemaCreateTableColumn];
+    $eventNames = [Events::onSomething, Events::onSomethingElse];
     $evm->addEventListener($eventNames, new MyEventListener());
-
-The following events are available.
```

Comment:
Do we usually remove docs for things that are deprecated? and if we're doing this, why do we keep the `Schema events` Section?

Reply:
> Do we usually remove docs for things that are deprecated?

Not always but I did that recently in https://github.com/doctrine/dbal/pull/5697 and https://github.com/doctrine/dbal/pull/5705. The documentation helps users use certain APIs but we don't want them to use them.

> why do we keep the `Schema events` Section?

My understanding is that besides describing the schema events, this section also describes how one could subscribe to an arbitrary event (which is still technically supported). This document will be entirely removed in a subsequent PR, so I don't think we need to restructure the document just before its removal.